src/content/configuration/dev-server.mdx

```diff
@@ -26,6 +26,8 @@ contributors:
 
 This page describes the options that affect the behavior of webpack-dev-server (short: dev-server) <Badge text="version >= 4.0.0" />. Migration guide from `v3` to `v4` can be found [here](https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md).
 
+W> <Badge text="webpack-dev-server >= v4.0.0" /> requires <Badge text="node >= v12.13.0" />, <Badge text="webpack >= v4.37.0" /> (but we recommend using <Badge text="webpack >= v5.0.0" />), and <Badge text="webpack-cli >= v4.7.0" />.
```

Comment:
Avoid using `<Badge />` a lot in those cases. It's a mess.

<img width="729" alt="CleanShot 2021-08-23 at 22 02 23@2x" src="https://user-images.githubusercontent.com/1091472/130460838-48054774-0d18-41d0-a21b-181577df442d.png">

Reply:
Is this good?

```suggestion
W> `webpack-dev-server >= v4.0.0` requires `node >= v12.13.0`, `webpack >= v4.37.0` (but we recommend using `webpack >= v5.0.0`), and `webpack-cli >= v4.7.0`.
```

Reply:
What about `webpack-dev-server 4.0.0+` instead of `webpack-dev-server >= v4.0.0`? It's much shorter.

Reply:
Btw, the `<Badge />` was introduced to indicate the available version for a new feature. So it's better to limit its usage. Otherwise it's distracting.